ui-admin/src/study/notifications/NotifcationConfigTypeDisplay.tsx

```diff
@@ -15,17 +15,17 @@ export const eventTypeDisplayMap: Record<string, string> = {
 }
 
 /** shows a summary of the notification config */
-export default function NotificationConfigTypeDisplay({ config }: {config?: NotificationConfig}) {
+export default function NotificationConfigTypeDisplay({ config }: {config?: Trigger}) {
```

Comment:
I noticed a lot of the UI components/variables still call them notification configs - are we keeping that way for continuity for users?

Reply:
yeah, I tried to get most of them in this PR, but I wanted to get this in quickly, and then we can clean up the stragglers as we go

Reply:
sounds good